<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="start_button">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stop_button">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="message_box">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>75</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Lucida Grande</family>
       </font>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:13pt;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>625</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSave">
     <property name="title">
      <string>Save..</string>
     </property>
     <addaction name="actionExperimental_Settings"/>
     <addaction name="actionExperimental_and_Instrument_Settings"/>
    </widget>
    <addaction name="menuSave"/>
    <addaction name="actionLoad"/>
   </widget>
   <widget class="QMenu" name="menuInstruments">
    <property name="title">
     <string>Instruments</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionHide_Message_Box"/>
    <addaction name="actionHide_Progress_Bar"/>
   </widget>
   <widget class="QMenu" name="menuSweep">
    <property name="title">
     <string>Sweep</string>
    </property>
    <addaction name="actionStart_Sweep"/>
    <addaction name="param_abort_sweep"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuInstruments"/>
   <addaction name="menuView"/>
   <addaction name="menuSweep"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExperimental_Settings">
   <property name="text">
    <string>Experimental Settings</string>
   </property>
  </action>
  <action name="actionExperimental_and_Instrument_Settings">
   <property name="text">
    <string>Experimental and Instrument Settings</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionHide_Message_Box">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide Message Box</string>
   </property>
  </action>
  <action name="actionHide_Progress_Bar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide Progress Bar</string>
   </property>
  </action>
  <action name="actionStart_Sweep">
   <property name="text">
    <string>Start Sweep</string>
   </property>
  </action>
  <action name="param_abort_sweep">
   <property name="text">
    <string>Abort Sweep</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionHide_Message_Box</sender>
   <signal>toggled(bool)</signal>
   <receiver>message_box</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>606</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
